configs/nginx/nginx.conf

```diff
@@ -0,0 +1,39 @@
+server {
+    listen 80;
+
+    # React
+    location / {
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header Host $host;
+        proxy_pass http://frontend:80;
+        proxy_redirect off;
+    }
+
+    # Django API
+    location /api/ {
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header Host $host;
+        proxy_pass http://api-dev:8000/api/;
+        proxy_redirect off;
+    }
+
+    # Django Admin
+    location /admin/ {
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header Host $host;
+        proxy_pass http://api-dev:8000/admin/;
+        proxy_redirect off;
+    }
+
+    # PGAdmin
+    location /pgadmin/ {
+        proxy_set_header X-Script-Name /pgadmin;
+        proxy_set_header Host $host;
+        proxy_pass http://pgadmin/;
+
+
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection 'upgrade';
+        proxy_cache_bypass $http_upgrade;
+    }
+}
```

Comment:
Check GitHub warnings. No newline at the end of the file 
https://stackoverflow.com/a/729795

Reply:
Thank you for feedback. Done